go/chat/flipmanager.go

```diff
@@ -532,6 +532,21 @@ func (m *FlipManager) StartFlip(ctx context.Context, uid gregor1.UID, hostConvID
 		return err
 	}
 
+	// Preserve the ephemeral lifetime from the conv/message to the game
+	// conversation.
+	if elf, err := utils.EphemeralLifetimeFromConv(ctx, m.G(), hostConv.Conv); err != nil {
```

Comment:
I think we need to just get the ephemeral lifetime of the message itself right?

Reply:
we don't have access to the message unless we change the commands interface to thread the message through instead of just the body text, what's the benefit of getting it from the message directly?